iib/web/models.py

```diff
@@ -816,10 +816,11 @@ def validate_graph_mode(graph_update_mode: Optional[str], index_image: Optional[
             )
         allowed_from_indexes: List[str] = current_app.config['IIB_GRAPH_MODE_INDEX_ALLOW_LIST']
         if index_image not in allowed_from_indexes:
-            raise Forbidden(
-                '"graph_update_mode" can only be used on the'
-                f' following index image: {allowed_from_indexes}'
-            )
+            if index_image is None or index_image.split(":")[0] not in allowed_from_indexes:
```

Comment:
Is not this creating backward incompatibility? 
I mean right now "users" are used to specify pull spec for index image and when they deploy this change all index images will work. That might be unexpected right?

Reply:
How do you mean all index images will work?  Can you give an example?
Old pull-spec (image:tag) will work as the condition on line 818 will result in True.

Reply:
Apologies, I overlooked the first condition. Makes sense, thank you!